charts/policy-reporter/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.ui.networkPolicy.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels: {{ include "policyreporter.labels" . | nindent 4 }}
+  name: {{ include "policyreporter.fullname" . }}
+spec:
+  podSelector:
+    matchLabels: {{- include "policyreporter.selectorLabels" . | nindent 6 }}
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - from:
+    ports:
+    - protocol: TCP
+      port: {{ .Values.global.port }} 
+  egress:
+  - to:
+    ports:
+    - protocol: TCP
+      port: {{ .Values.ui.service.port }} 
```

Comment:
It would be great if you can add an value to add additional egress rules for additional targets like Loki as example

```yaml
networkPolicy:
  egress:
  - to:
    - namespaceSelector:
        matchLabels:
          kubernetes.io/metadata.name: loki-stack
      podSelector:
        matchLabels:
          app: loki
    ports:
    - port: 3100
      protocol: TCP
```